### PR TITLE
Minimal verbose mode

### LINKED ISSUE #993: 🙋Add verbose logging mode
**Choose one:** is this a 🐛 bug report or 🙋 feature request?

🙋 feature request

It's impossible to debug an issue or to show anyone if I can't see what's happening. All logging messages are replaced one the build is done.

### 🎛 Configuration (.babelrc, package.json, cli command)

```sh
parcel watch
```

### 🤔 Expected Behavior

```sh
$ parcel watch --verbose
Server running at http://localhost:1234 
⏳  Building index.html...
⏳  Building index.js...
✨  Built in 3.78s.
⏳  Building index.js...
✨  Built in 0.78s.
⏳  Building index.html...
✨  Built in 0.1s.
```

### 😯 Current Behavior

```sh
$ parcel watch
Server running at http://localhost:1234 
✨  Built in 3.78s.
# what happened?! what did it do?
```

### 🔦 Context

Mostly for debugging purposes

### 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           | 1.6.2
| Node             | 8.6.0
| npm/Yarn         | 5.7.1
| Operating System | macOS 10.13.3

### PR DESCRIPTION
**Pull request by DeMoorJasper**
Adds verbose and debug mode:

4: verbose, keep everything in log with timestamps, instead of using statusspinner
5: debug, save everything to a file with timestamps

Shouldn't slow down anything, verbose might even speed everything up as it uses native console.log instead of a spinner.

Also adds a log call after an asset has been processed to track which files are being successfully built

Closes #993

### CODE REVIEW

File: src/cli.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ program
   .option(
     '--log-level <level>',
     'set the log level, either "0" (no output), "1" (errors), "2" (warnings + errors) or "3" (all).',
```
Comment (devongovett) [inline src/cli.js]: Should we update this message to add the new verbose modes?

Comment (DeMoorJasper) [reply src/cli.js]: Probably, updated it to include the verbose and debug mode

File: src/Logger.js [hunk 2]
```diff
@@ -58,6 +64,28 @@ class Logger {
...
+    message = `[${currDate.toLocaleTimeString()}]: ${message}`;
+    if (this.logLevel > 4) {
+      if (!this.logFile) {
+        this.logFile = fs.createWriteStream(
+          path.join(
+            process.cwd(),
+            `parcel-debug-${currDate.toLocaleDateString()}@${currDate.toLocaleTimeString()}.log`
+          )
+        );
+        this.logFile.write = this.logFile.write.bind(this.logFile);
```
Comment (devongovett) [inline src/Logger.js]: is this bind needed?

Comment (DeMoorJasper) [reply src/Logger.js]: Don't think so, I'll remove it, seems to work fine without

**Pull request merged**